src/constants/routing.ts

```diff
@@ -1,6 +1,7 @@
 // a list of tokens by chain
 import { Currency, Token } from '@uniswap/sdk-core'
 import { SupportedChainId } from './chains'
+import 'polyfill-object.fromentries'
```

Comment:
I'd rather see this in a polyfill file, as it doesn't import anything obviously used in this file. That way, it will be easier to track polyfills in the future.

Reply:
I have create a polyfill file in `/src` and import it in `index.tsx` now. Is this appropriate ?